src/client.rs

```diff
@@ -59,12 +82,28 @@ impl ClientBuilder {
     /// 49.7 days).
     #[must_use]
     pub fn requested_session_timeout(mut self, requested_session_timeout: Duration) -> Self {
-        let config = unsafe { UA_Client_getConfig(self.0.as_mut_ptr()).as_mut() };
-
-        config.unwrap().requestedSessionTimeout =
+        self.config_mut().requestedSessionTimeout =
             u32::try_from(requested_session_timeout.as_millis())
-                .expect("secure channel life time should be in range of u32");
+                .expect("secure channel life time (in milliseconds) should be in range of u32");
+        self
+    }
 
+    /// Sets connectivity check interval.
+    ///
+    /// Use `None` to disable background task.
+    ///
+    /// # Panics
+    ///
+    /// The given duration must be non-negative and less than 4,294,967,295 milliseconds (less than
+    /// 49.7 days).
+    #[must_use]
+    pub fn connectivity_check_interval(
+        mut self,
+        connectivity_check_interval: Option<Duration>,
```

Comment:
Why does this method allow to pass `Option<Duration>` for a special case while the other _timeout_ methods always require a valid duration?

Reply:
Then we should probably mention that both `None` and `Some(Duration::EMPTY)` have the same effect?

Reply:
The docs contain recommendations, although they don't cover all possible cases exhaustively. We could leave it as is.